examples/consuming_packages/versioning/conflicts/force/conanfile.py

```diff
@@ -0,0 +1,4 @@
+from conan import ConanFile
```

Comment:
I could not see the usage of this file in the sample execution. Is this intentional?

Reply:
Nop, good catch.